config/web.php

```diff
@@ -16,6 +16,9 @@
             $params['yiisoft/session']['session']['options'],
             $params['yiisoft/session']['session']['handler'],
         ],
+        'reset' => function () {
+            $this->destroy();
```

Comment:
Wouldn't that result in losing session data between requests?

Reply:
$this->clear() should be here instead?